docs/runbooks/CnaoNmstateMigration.md

```diff
@@ -0,0 +1,14 @@
+# CnaoNmstateMigration
+
+## Meaning
+
+Cluster-network-addons-operator (CNAO) will stop support of kubernetes-nmstate deployment.
+This alert fires when nmstate is deployed, but there is no Kubernetes NMState Operator installed.
+
+## Impact
+
+Having kubernetes-nmstate deployed via CNAO will block future upgrade.
+
+## Mitigation
+
+Please, install standalone Kubernetes NMState Operator. After Kubernetes NMState Operator is installed, CNAO will hand over the kubernetes-nmstate deployment to the NMState Operator automatically.
```

Comment:
This is not true, right ? for >= 4.11

Reply:
This alert doesn't exist on 4.11 (or cnao release-0.76 and later).
Also, since this is an u/s project, I'm not sure if we should refer to d/s versions, which is why I decided to remove references to versions in this [change](https://github.com/kubevirt/monitoring/compare/e1a49318dc378ed4868f374d4541c838486336a8..bcf1a7f0ba3e7c15d017ed89d9333df1db3cc479).

Since we're blocking upgrade from 4.10, we can assume that on 4.11, users either don't use knmstate or have already migrated, which is why there's no need to have this alert in 4.11.